ide/idea-iml-file.xml

```diff
@@ -49,25 +49,27 @@
         <orderEntry type="module-library">
             <library>
                 <CLASSES>
-                    <root url="file://$MODULE_DIR$/lib" />
+                    <root url="file://$MODULE_DIR$/build/lib/jars" />
                 </CLASSES>
```

Comment:
do we need both `<CLASSES>` sections? (only jar files are found in `build/lib/jars` and `build/test/lib/jars`)

Reply:
perhaps we do not, I've just taken the file generated by IntelliJ and those entries were there. I'll try if we can safely remove them

Reply:
ok, unfortunately we cannot remove `CLASSES` entries, IntelliJ would recognise it as an empty library :(